lib/process-services-cloud/src/lib/form/components/form-cloud.component.html

```diff
@@ -3,47 +3,82 @@
     </ng-content>
 </div>
 
-<div *ngIf="hasForm()" class="adf-form-container">
-    <mat-card class="adf-form-container-card">
-        <mat-card-header *ngIf="showTitle || showRefreshButton || showValidationIcon">
-            <mat-card-title>
-                <h4>
-                    <div *ngIf="showValidationIcon" class="adf-form-validation-button">
-                        <i id="adf-valid-form-icon" class="material-icons"
-                            *ngIf="form.isValid; else no_valid_form">check_circle</i>
-                        <ng-template #no_valid_form>
-                            <i id="adf-invalid-form-icon" class="material-icons adf-invalid-color">error</i>
-                        </ng-template>
-                    </div>
-                    <div *ngIf="showRefreshButton" class="adf-form-reload-button">
-                        <button mat-icon-button (click)="onRefreshClicked()">
-                            <mat-icon>refresh</mat-icon>
-                        </button>
-                    </div>
-                    <span *ngIf="isTitleEnabled()" class="adf-form-title" [matTooltip]="form.taskName">
-                        {{form.taskName}}
-                        <ng-container *ngIf="!form.taskName">
-                            {{'FORM.FORM_RENDERER.NAMELESS_TASK' | translate}}
-                        </ng-container>
-                    </span>
+<div *ngIf="hasForm()" class="adf-cloud-form-container"
+    [class.adf-cloud-form-overlay-container]="overlayMode"
+    [class.adf-cloud-form-inline-container]="!overlayMode">
+    <div class="adf-cloud-form-content"
```

Comment:
I understand that for the sake of POC, we are doing some css magic, but wouldn't it be better to use Angular Material overlaying?

Reply:
I have done it the same way it is done in the ADF viewer, but I can explore this option